### ISSUE TITLE
Is Proxy supported ?

### ISSUE THREAD

**Issue by g8up** [fent/node-ytdl#84]
I'm from China.
I can't visit youtube without proxy.

**Comment by mh4ck** [comment]
Yes you can add an [Https Proxy Agent](https://www.npmjs.com/package/https-proxy-agent) by yourself via options.

**Comment by ghost** [comment]
@mh4ck I don't really know JS that much would you please elaborate? How about SOCKS5?
